native/packages/react-native-bpk-component-badge/src/BpkBadge-test.common.js

```diff
@@ -19,14 +19,26 @@
 /* @flow */
 
 import React from 'react';
-import { View } from 'react-native';
+import { View, StyleSheet } from 'react-native';
 import renderer from 'react-test-renderer';
-import { spacingSm } from 'bpk-tokens/tokens/base.react.native';
+import { spacingSm, colorWhite } from 'bpk-tokens/tokens/base.react.native';
+import BpkIcon, { icons } from 'react-native-bpk-component-icon';
+import { capitalize } from 'lodash';
 
 import BpkBadge, { BADGE_TYPES, BADGE_DOCKED_TYPES } from './BpkBadge';
 
+const style = StyleSheet.create({
+  iconDestructive: {
+    color: colorWhite,
+  },
+  iconOutline: {
+    color: colorWhite,
+  },
+});
+
```

Comment:
These styles will be unnecessary once we change it to automatically colour the accessory view.